### PR TITLE
Add support for mongoexport and mongoimport

### PR DESCRIPTION
**Pull request by hoang1417**

### CODE REVIEW

**Comment by dblock** [comment]
Great! This needs tests and a CHANGELOG entry, please.

**Comment by hoang1417** [comment]
Sure, I'll try to add them soon and notify you later. Thanks for looking at it.

**Comment by dblock** [comment]
Bump @hoang1417

**Comment by hoang1417** [comment]
Hi @dblock, sorry it's too late from my promise :sweat:. 

However, finally I can spend time to finish it now, please review and let me know if I need to update anything. :smiley: 

Thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-0.4.1 (Next)
+0.4.2 (Next)
 ============
 
 * Your contribution here.
 
+0.4.1 (10/23/2015)
```
Comment (dblock) [inline CHANGELOG.md]: This should be under next, and you can 1-line them, like `Added support for x and y - ...`.

**Comment by dblock** [comment]
Looks great. Can you please squash your commits and fix-up that CHANGELOG. Thx,

**Comment by hoang1417** [comment]
Sure, I 1-lined them :). 

About "squash your commits", can you explain more detailed what you're expecting? Thanks.

**Comment by dblock** [comment]
Squashing means making all these commits into 1 commit.

This could be helpful: http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html

**Comment by hoang1417** [comment]
Hi @dblock, yeah now I see your point. First thing is my pull request should be from a **feature** branch and next is my commits look messy. I created new pull for this, thanks for your feedback. :+1: 

I read your post and in my case I can just checkout a new branch and `reset HEAD~8 --soft` to squash them but the `squash` option is new to me. Thanks again for the info.

**Pull request closed without merging**